CHANGELOG.rst

```diff
@@ -18,6 +18,7 @@ Changed
 
 Removed
 -------
+- removed possibility to execute ``python -m rasa_core.train`` etc. (e.g. scripts in ``rasa.core`` and ``rasa.nlu``). Use the CLI for rasa instead, e.g. `rasa train core``.
```

Comment:
Please also add a note to docs/migration with instructions on how to go from old calls of these scripts to the new cli

Reply:
can you please point me to the docs/migration where I can add the instructions? thanks.

Reply:
https://github.com/RasaHQ/rasa_nlu/blob/master/docs/core/migrations.rst ?